res/database/Default/database_programmes.xml

```diff
@@ -11514,10 +11512,9012 @@ Dream scenes and sentiments warp the storyline.</en>
+				<member index="3" function="2">3fcd8263-9bf4-4c23-9323-0e4105d75dbb</member>
+				<member index="4" function="2">1c82f854-c349-4dc4-aa15-7cda8594d803</member>
+				<member index="5" function="1024">a7aa2d24-06d8-4053-83e8-288ac64ea7b6</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="I/D/F" distribution="1" maingenre="4" subgenre="7,14" flags="" blocks="3" price_mod="0.5" />
+			<releaseTime year="1986" day="9" />
+			<ratings critics="71" speed="80" outcome="41" />
+		</programme>
+		<programme id="Stukov_Das Ding aus einer anderen Welt" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0084787" creator="9584" created_by="Stukov" comment="Alt description: Es ist die erste Woche im Winter des Jahres 1982. Eine Forschungsstation in der Antarktis bekommt Besuch von einer außerirdischen Kraft, die alles assimilieren kann, was sie berührt.">
+			<title>
+				<de>Das Etwas aus der Fremde</de>
+				<en>The What</en>
+			</title>
+			<title_original>
+				<de>Das Ding aus einer anderen Welt</de>
+				<en>The Thing</en>
+			</title_original>
+			<description>
+				<de>Eine Forschungsstation in der Antarktis. Seltsames Verhalten des Personals. Plötzlich mutiert ein Hund zu einem Monster. Achso, da ist seit tausenden von Jahren ein Raumschiff im Eis. Na dann ist das ja geklärt.</de>
+				<en>A research station in Antarctica. Weird behavior of the staff. Suddenly a dog mutates into a monster. Oh, there's been a spaceship in the ice for thousands of years. Well, that settles that.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">1ae56fa2-8710-48d9-acc5-86bc250c1de0</member>
+				<member index="1" function="2">91af6b99-191b-41e2-8c41-f7c0498e1c18</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="USA" distribution="1" maingenre="12" subgenre="14" flags="" blocks="2" price_mod="0.96" />
+			<releaseTime year="1982" day="6" />
+			<ratings critics="77" speed="85" outcome="36" />
+		</programme>
+
+		<!-- added by scr0llbaer -->
+
+		<programme id="697bc87c-ec29-420d-9489-f50cfea2a0d7" product="1" licence_type="1" creator="9551" imdb_id="tt0092563">
+			<title>
+				<de>Engelsorgan</de>
+				<en>Angel Organ</en>
+			</title>
+			<title_original>
+				<de>Angel Heart</de>
+				<en>Angel Heart</en>
+			</title_original>
+			<description>
+				<de>Stilbildender Meisterfilm von [0|Full], spielt in den späten 1950er Jahren tief im Voodoo-Viertel von New Orleans und handelt von einem Privatdetektiv, der in eine Reihe von brutalen Morden verwickelt wird.</de>
+				<en>Style-defining masterpiece by [0|Full] set in the late 1950s deep in Orlean's voodoo district about a private investigator who becomes embroiled in a series of brutal murders.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">21cf3fe5-98dc-48a0-b84f-a31c54ba2861</member> <!-- TODO: function="5" -->
```

Comment:
combined flags in programmes are now supported

Reply:
ok I'll incorporate all of these.